Sofa/framework/Simulation/Core/src/sofa/simulation/BaseSimulationExporter.cpp

```diff
@@ -95,6 +95,14 @@ void BaseSimulationExporter::handleEvent(Event *event){
             }
         }
     }
+    else if (AnimateBeginEvent::checkEventType(event))
+    {
+        if (firstStep && d_isEnabled.getValue() && d_exportAtBegin.getValue())
```

Comment:
I am a bit concerned because it means you have to press the step button to export at the initialization.

Reply:
I guess what we want is an event triggered after all objects are initialized and before the beginning of the animation. I don't think there is such an event other than `bwdInit`...

Reply:
As the data that trig this condition is called "exportAtBegin" I wouldn't have didn't expected anything special without pressing "step/animate", so to me the change done by Hugo seems coherent at first sight

Reply:
It is still a change of behavior and it should be addressed. Beside, the Data description is `export file at the initialization`

Reply:
Additional comment for the record, dumping after "init" but before "animate" is not correct if implemented in bwdInit() as bwdInit() is "in between the init" so there is no garantee that all component have executed their init + bwdInit.

Adding a new event to indicate "onLoaded" and trigger component's actions is one solution to that.